### ISSUE TITLE
Time LIMIT

### ISSUE THREAD

**Issue by mwiinga** [wolly66/woo-wolly-one-time-buy#1]
I would like to have my products bought once every 30 days.
Is it possible with this plugin?

**Comment by wolly66** [comment]
No, not without modify the plugin

**Comment by mwiinga** [comment]
..unfortunately I do not know how to write code.
Would you help please??...& just direct where to past it??

**Comment by wolly66** [comment]
Not now, and I cannot promise you that I'll do.

Maybe in the future

**Comment by mwiinga** [comment]
Oh okay 
Will wait on you. 
Thank you
